File: XamarinSportApp-master/Web/Fanword/Fanword.Business/ViewModels/Events/EventFilter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Fanword.Business.ViewModels.Events
{
    public class EventFilter
    {
        public string SportId { get; set; }
        public string TeamId { get; set; }
        public string Name { get; set; }
        public string StartDate { get; set; }
        public string EndDate { get; set; }
        public DateTime? StartDateUtc { get; set; }
        public DateTime? EndDateUtc { get; set; }
        public int currentPage { get; set; }

    }
}

File: XamarinSportApp-master/Web/Fanword/Fanword.Web/TypeScripts/PocoModels/EventFilter.ts
class EventFilter {
    sportId: string;
    teamId: string;
    name: string;
    startDate: string;
    endDate: string;
    startDateUtc: Date;
    endDateUtc: Date;
    currentPage: Number;
    totalPages: Number;
    totalEvents: Number;
    constructor() {
        this.startDateUtc = null
        this.endDateUtc = null;
        this.sportId = "";
        this.teamId = "";
        this.startDate = "";
        this.endDate = "";
    }
}